docs/requirements.txt

```diff
@@ -3,3 +3,4 @@ boto3
 deprecation
 sphinx-rtd-theme
 sphinx
+urllib3
```

Comment:
I don't think this is necessary for building the document.

Reply:
Fixed by https://github.com/pfnet/pfio/pull/282/commits/bbe74b6df4d7b484b564dd90c5fb548bbc04d7bf